dvl_gazebo/models/teledyne_whn/model.sdf

```diff
@@ -0,0 +1,369 @@
+          <effort>0</effort>
+          <velocity>0</velocity>
+        </limit>
+      </axis>
+    </joint>
+
+    <!-- Single sonar link 3 -->
+    <link name="dvl_sonar3_link">
+      <inertial>
+        <pose>0 0 0 0 0 0</pose>
+        <mass>0.001</mass>>
+        <inertia>
+          <ixx>0.000000017</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.000000017</iyy>
+          <iyz>0</iyz>
+          <izz>0.000000017</izz>
+        </inertia>
+      </inertial>
+      <collision name="sonar3_collision">
+        <pose>0 0 0 0 0 0</pose>
+        <geometry>
+          <cylinder>
+            <radius>0.000001</radius>
+            <length>0.000001</length> 
+          </cylinder>
+        </geometry>
+      </collision>
+
+      <sensor type="ray" name="dvl_sonar3">
+        <pose>0 0 0 0 0 -0.52</pose>
+        <update_rate>10</update_rate>
+        <visualize>true</visualize>
+        <ray>
+          <scan>
+            <horizontal>
+              <samples>3</samples>
+              <resolution>1</resolution>
+              <min_angle>-0.03</min_angle>
+              <max_angle>0.03</max_angle>
+            </horizontal>
+            <vertical>
+              <samples>3</samples>
+              <resolution>1</resolution>
+              <min_angle>-0.03</min_angle>
+              <max_angle>0.03</max_angle>
+            </vertical>
+          </scan>
+          <range>
+            <min>0.55</min>
+            <max>81</max>
+            <resolution>0.01</resolution>
+          </range>
+        </ray>
+
+        <plugin filename="libgazebo_ros_range.so" name="dvl_sonar3_plugin">
+          <gaussianNoise>0.005</gaussianNoise>
+          <alwaysOn>true</alwaysOn>
+          <updateRate>10</updateRate>
+          <topicName>dvl_ray_sonar3</topicName>
+          <frameName>/dvl_sonar3_link</frameName>
+          <fov>0.06</fov>
+          <radiation>ultrasound</radiation>
+        </plugin>
+      </sensor>
+    </link>
+
+    <joint name="dvl_sonar3_joint" type="fixed">
+      <pose>0 0 0 0 0 0</pose>
+      <parent>dvl_base_link</parent>
+      <child>dvl_sonar3_link</child>
+      <axis>
+        <xyz>1 0 0</xyz>
+        <limit>
+          <upper>0</upper>
+          <lower>0</lower>
+          <effort>0</effort>
+          <velocity>0</velocity>
+        </limit>
+      </axis>
+    </joint>
+
+    <plugin filename="libuuv_gazebo_ros_dvl_plugin.so" name="libuuv_gazebo_dvl_plugin"> -->
+      <robot_namespace>dvl</robot_namespace> <!-- (string, rqd): ros namespace in which the messages are published -->
+      <link_name>dvl_base_link</link_name>       <!-- (string, rqd): name of the body which holds the IMU sensor -->
+      <sensor_topic>dvl</sensor_topic>       <!-- (string): name of the sensor output topic and prefix of service names -->
+      <update_rate>10</update_rate>          <!-- Update period [s] -->
+      <noise_sigma>0.1</noise_sigma>         <!-- std dev of position estimates in x, y, z [m/s] -->
```

Comment:
Is this std dev for position or for velocity?  If memory serves the std dev for velocity is something like 3 mm/s - it is on the spec. sheet.

Reply:
I wanted to get something working and posted, but I'll need to go back through everything to match the published specs before we do any evaluation beyond figuring out how it works and what it's doing.

Reply:
Sounds good.